mpmath/libmp/libmpf.py

```diff
@@ -1645,7 +1677,7 @@ def format_mpf(num, format_spec):
                 rpad*format_dict['fill_char']
 
     return lpad*format_dict['fill_char'] + sign + digits \
-            + rpad*format_dict['fill_char']
+        + rpad*format_dict['fill_char']
```

Comment:
```suggestion
            + rpad*format_dict['fill_char']
```
Lets avoid unrelated changes.